src/app/components/forum/topic-list/topic-list.html

```diff
@@ -3,11 +3,11 @@
 	<div class="forum-topic-list-item"
 		v-for="topic of topics"
 		:key="topic.id"
-		:class="{ '-has-voting': useUpvotes }"
+		:class="{ '-has-voting': useUpvotes && !topic.is_locked }"
```

Comment:
may as well use a getter in the component class: `shouldShowVoting` since we do this multiple times